controller/test/sensor_tests/SensorTests.cpp

```diff
@@ -157,11 +158,48 @@ TEST(SensorTests, FullScaleReading) {
   }
 }
 
-//@TODO: Write complete unit tests for the volumetric flow calculation
-TEST(SensorTests, TestVolumetricFlowCalculation) {
-  sensors_init(PressureSensors::DEFAULT_VENTURI_PORT_DIAM,
-               PressureSensors::DEFAULT_VENTURI_CHOKE_DIAM);
-  float volumFlow = convert_diff_pressure_to_volumetric_flow(1.0f);
+// These tests expect Venturi Diamters of 14 and 5.5 mm.
+// If the Default PressureSensors::DEFAULT_VENTURI_PORT_DIAM and
+// DEFAULT_VENTURI_CHOKE_DIAM are changed from this, the tests will fail unless
+// you update the expected values accordingly.
+TEST(SensorTests, TestPositiveVolumetricFlowCalculation) {
+  sensors_init();
```

Comment:
If you want you could do something like the following, makes the tests a little less repetitive.

```
class SensorTests {
  public:
    void SetUp() { sensors_init(); }
};
TEST_F(SensorsTest, blah) {
  EXPECT_NEAR(pressure_delta_to_volumetric_flow(-1.0f), -9.52e-4f, COMPARISON_TOLERANCE_FLOW);
}
```

nbd, up to you.

Reply:
Good to know, I'll read up more on gtest sometime. I'll leave it for another time to cleanup the entire test module for simplicity.